src/main/java/com/ita/if103java/ims/mapper/jdbc/WarehouseWidgetRowMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.ita.if103java.ims.mapper.jdbc;
+
+import com.ita.if103java.ims.dto.WarehouseWidgetDto;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Component
+public class WarehouseWidgetRowMapper implements RowMapper<WarehouseWidgetDto> {
+    @Override
+    public WarehouseWidgetDto mapRow(ResultSet resultSet, int rowNum) throws SQLException {
+        WarehouseWidgetDto warehouseWidgetDto = new WarehouseWidgetDto();
+        warehouseWidgetDto.setWarahouseCapacity(resultSet.getLong("warehouse_top_id"));
+        warehouseWidgetDto.setWarehouseLoad(resultSet.getLong("warehouse_id"));
```

Comment:
Not sure if this work, because I do not see such columns in your query.

Please add aliases in the query and use them here, like
`SELECT wh.top_warehouse_id, sum(wh.capacity) AS capacity, sum(quantity*volume) AS load`
And then here you can use
```
warehouseWidgetDto.setId(resultSet.getLong("top_warehouse_id"));
warehouseWidgetDto.setCapacity(resultSet.getLong("capacity"));
warehouseWidgetDto.setLoad(resultSet.getLong("load"));
```